extensions/amp-base-carousel/1.0/storybook/Basic.amp.js

```diff
@@ -42,4 +42,30 @@ storiesOf('amp-base-carousel', module)
         ))}
       </amp-base-carousel>
     );
+  })
+  .add('customize button CSS', () => {
+    return (
+      <>
+        <style jsx global>
+          {`
+            amp-base-carousel::part(arrow next) {
```

Comment:
Do you want to add a storybook for Preact that uses regular selectors to achieve the same goal?